CHANGELOG.md

```diff
@@ -3,6 +3,14 @@
 [PyPI History][1]
 [1]: https://pypi.org/project/demisto-py/#history
 
+## Unreleased
+
+* Changed package management from pip to Poetry for better dependency handling and packaging.
+* Replaced `tox` with `Poetry` to run `pytest`.
+* Changed the publish process to PyPI from using `twine` to `Poetry`.
+* Update the README.md to reflect the new changes.
+* Moved the handling of `Poetry` publish to `CircleCI`.
```

Comment:
```suggestion
```
This is mostly for changes that affect the python library, not infrastructure ones